oem/firstvoices/android/build.gradle

```diff
@@ -6,8 +6,8 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.4.2'
-        classpath 'io.sentry:sentry-android-gradle-plugin:2.1.2'
+        classpath 'com.android.tools.build:gradle:8.9.2'
```

Comment:
I assume it's correct that we're using Gradle 8.12 but here it's 8.9.2 !?

Reply:
yeah, it's always confusing that the Gradle version is different from the Gradle tools plugin version.

Android Studio prompts the latest version of the tool plugin